apitrackingmore.py

```diff
@@ -96,24 +96,30 @@ def formato_obj(json, carrier, code, retries):
             return status.NOT_FOUND_TM
     mensagem = ''
     for evento in reversed(tabela):
+        codigo_novo = None
+        msg_codigo_novo = None
         try:
             data = datetime.strptime(evento['Date'], '%Y-%m-%d %H:%M:%S').strftime("%d/%m/%Y %H:%M")
         except ValueError:
             data = datetime.strptime(evento['Date'], '%Y-%m-%d %H:%M').strftime("%d/%m/%Y %H:%M")
         situacao = evento['StatusDescription']
         observacao = evento['checkpoint_status']
         try:
-            observacao = 'Código novo: <code>' + geartrack.getcorreioscode(carrier, code) + '</code>'
+            codigo_novo = geartrack.getcorreioscode(carrier['code'], code)
+            msg_codigo_novo = ('Data: {}' +
+                '\nNovo código do pacote: <code>{}</code>'
+            ).format(data, codigo_novo)
         except:
             pass
         mensagem = ('Data: {}' +
             '\nSituacao: <b>{}</b>' +
             '\nObservação: {}'
         ).format(data, situacao, observacao)
         stats.append(mensagem)
+        if codigo_novo and str(codigo_novo) not in str(stats) and msg_codigo_novo != None:
```

Comment:
Não seria melhor verificar se a variavel é None com `msg_codigo_novo is not None`?